src/game.js

```diff
@@ -0,0 +1,113 @@
+function Game(frame) {
+  this.frame = new Frame();
+  this.firstRoll = 0;
+  this.secondRoll = 0;
+  this.thirdRoll = 0;
+  this._frames = {};
+}
+
+Game.prototype.getGameInfo = function () {
+  return this._frames;
+};
+
+Game.prototype.storeFrame = function () {
```

Comment:
This function has too many statements. (19)<br>This function's cyclomatic complexity is too high. (8)